scripts/Muon/GUI/Common/plot_widget/base_pane/base_pane_presenter.py

```diff
@@ -83,7 +83,12 @@ def handle_workspace_deleted_from_ads(self, workspace: Workspace2D):
         Handles a workspace being deleted from ads by removing the workspace from the plot
         :param workspace: workspace 2D object
         """
-        workspace_name = workspace.name()
+        workspace_name = None
```

Comment:
Just a small question, why do we need this check here? And why not the same check above at line 76?

Reply:
This is to stop errors when having both FDA and MA open at the same time. For some reason the string of the workspace gets sent to here (even when its not deleted).

Reply:
Could you add a comment to explain this for the future? maybe then someone might be able to find the root cause rather than this patch fix

Reply:
```suggestion
        # this is to allow both MA and FDA to be open at the same time
        # without this an error is thrown as the ws name is passed as a string
        workspace_name = None
```

Reply:
Can we create an issue to investigate this? It feels like something we don't understand that likely means a it's going to come up in more places and cause troubles?

Reply:
I will add it to this issue https://github.com/mantidproject/mantid/issues/31435 I think they are probably linked/related/similar